test/integration/patients/worklist/schedule.js

```diff
@@ -955,11 +1123,40 @@ context('schedule page', function() {
       .should('contain', 'Test Action');
   });
 
+  specify('reduced schedule clinician - no groups', function() {
+    cy
+      .routeCurrentClinician(fx => {
+        fx.data.id = '123456';
+        fx.data.attributes.enabled = true;
+        fx.data.relationships.role.data.id = '44444';
+        fx.data.relationships.groups.data = [];
+        return fx;
+      })
+      .routeActions()
+      .routeAction()
+      .routePatientByAction()
+      .routePatient()
+      .visit('/')
+      .wait('@routeActions')
+      .itsUrl()
+      .its('search')
+      .should('not.contain', 'filter[group]');
+
+    cy
+      .url()
+      .should('contain', 'schedule');
+
+    cy
+      .get('[data-list-region]')
+      .find('.schedule-list__list-row')
+      .should('have.length.above', 0);
+  });
```

Comment:
This loads the reduced schedule where the clinician has no groups. It checks that the api request doesn't include the `filter[group]` filter in the request and that the list still successfully displays actions.